common/Tests/Framework/src/Assert.ts

```diff
@@ -19,7 +21,7 @@ export class Assert {
     * @param message A short description of the assertion
     */
     public static deepEqual(expected: any, actual: any, message?: string): any {
-        return QUnit.assert.deepEqual(actual, expected, message);
+        return QUnit.assert.deepEqual(actual, expected, message || expectedToString(expected));
```

Comment:
Just a test update to try and get some more automatic information as part of the assertion log so you may not always need to add a message to the assertion.